### PR TITLE
setup: bump gcsfs

### PR DESCRIPTION
**Pull request by pared**
Related: #7053

* [ ] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist.

* [ ] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

Fixes dependency inconsistencies which are failing the build: https://github.com/iterative/dvc/commit/ea49fb95f7a493dc888751b8343777df51951871/checks

### CODE REVIEW

**Comment by efiop** [comment]
Oops, my bad, I clicked `commit` instead of creating a PR, sorry 🙁

File: setup.cfg [hunk 1]
```diff
@@ -98,7 +98,7 @@ azure =
     azure-identity>=1.4.0
     knack
 gdrive = pydrive2[fsspec]>=1.9.4
-gs = gcsfs==2021.11.0
+gs = gcsfs==2021.11.1
```
Comment (efiop) [inline setup.cfg:101]: While we are here, let's unpin gcsfs and ossfs and s3fs, since everything else is also unpinned anyway.

**Comment by skshetry** [comment]
I think we agreed not to bump versions for filesystems' dependencies.

**Comment by efiop** [comment]
For the record: benchmark failures are being fixed in https://github.com/iterative/dvc-bench/pull/309

**Pull request merged**